src/MyApplication.cpp

```diff
@@ -0,0 +1,91 @@
+#include <Magnum/GL/DefaultFramebuffer.h>
+#include <Magnum/Platform/GLContext.h>
+
+#include <wx/app.h>
+#include <wx/frame.h>
+#include <wx/glcanvas.h>
+#include <wx/sizer.h>
+#include <wx/version.h>
+
+using namespace Magnum;
+
+class MyApplication: public wxApp {
+    public:
+        bool OnInit();
+};
+
+class MainFrame: public wxFrame {
+    public:
+        MainFrame(int argc, char** argv);
+        ~MainFrame();
+
+    private:
+        void OnPaint(wxPaintEvent& event);
+
+        wxGLCanvas* _wxGlCanvas;
+        wxGLContext* _wxGlContext;
+        Platform::GLContext _glContext;
+};
+
+wxIMPLEMENT_APP(MyApplication);
+
+bool MyApplication::OnInit() {
+    MainFrame *frame = new MainFrame{argc, argv};
+    frame->Show(true);
+    return true;
+}
+
+MainFrame::MainFrame(int argc, char** argv): wxFrame{nullptr, wxID_ANY, "Magnum wxWidgets Application"}, _glContext{NoCreate, argc, argv} {
+    wxBoxSizer* bSizer;
+    bSizer = new wxBoxSizer{wxVERTICAL};
+
+#if (wxMAJOR_VERSION == 3) && (wxMINOR_VERSION >= 1)
```

Comment:
I'm always indenting the preprocessor. (I can also do the change while merging, no worries.)

Reply:
Indented locally. Will push soon with the other changes.